RELEASE.md

```diff
@@ -44,6 +44,7 @@
 *   Added custom config to Transform component and made it available to
     pre-processing fn.
 *   Supported custom extractors in Evaluator.
+*   Supported `Date` spec in input config for file based ExampleGen.
```

Comment:
mention the type is changed in breaking change

Reply:
updated, thanks!

Reply:
I didn't find anything added under '## Breaking changes' section?

Reply:
@ruoyu90 we decided to revert this type change. the proposed change was making span and version int custom properties, but since that would likely block the CL, we've decided to push it back.

https://github.com/tensorflow/tfx/pull/2285#discussion_r469564944